src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs

```diff
@@ -20,7 +20,7 @@ internal class PickMembersDialogViewModel : AbstractNotifyPropertyChanged
             IGlyphService glyphService,
             ImmutableArray<ISymbol> members)
         {
-            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).OrderBy(s => s.MemberName).ToList();
+            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).ToList();
```

Comment:
That way you see all of 'objects' members grouped together, then the next highest derived type, etc. etc.